### ISSUE TITLE
error in backend with current master

### ISSUE THREAD

**Issue by DavidBruchmann** [pluspol-interactive/templavoilaplus#282]
```
Fatal error: Only variables can be passed by reference in typo3conf/ext/templavoilaplus/Classes/Configuration/BackendConfiguration.php on line 45
```

**Comment by DavidBruchmann** [comment]
Tried with PHP 7.2 and 7.4

**Comment by opi99** [comment]
Current master generally does not work yet in backend.

**Comment by opi99** [comment]
Current master generally does not work yet in backend.

**Comment by opi99** [comment]
Was fixed by 083e3850 forgot to reference it corretcly.
